package.json

```diff
@@ -50,10 +50,11 @@
     "@balena/jellyfish-core": "^14.4.2",
     "@balena/jellyfish-jellyscript": "^5.1.40",
     "@balena/jellyfish-logger": "^4.0.38",
-    "@balena/jellyfish-queue": "^4.0.48",
+    "@balena/jellyfish-queue": "4.1.0-joshbwlng-scheduled-actions-be9e13eb72be83bb02b94db2435b295fa3bc6525",
```

Comment:
Obviously updating this when https://github.com/product-os/jellyfish-queue/pull/1059 is merged